tasks/main.yaml

```diff
@@ -3,6 +3,7 @@
   group:
     name: '{{ zookeeper_group }}'
     state: present
+  when: zookeeper_create_user_group|bool == true
```

Comment:
Linting issue: "Don't compare to literal True/False".

Similar change required in other lines.